stats/stats-server/src/server.rs

```diff
@@ -145,8 +147,19 @@ pub async fn stats(mut settings: Settings) -> Result<(), anyhow::Error> {
         metrics::initialize_metrics(charts.charts_info.keys().map(|f| f.as_str()));
     }
 
-    let read_service =
-        Arc::new(ReadService::new(db, blockscout, charts, settings.limits.into()).await?);
+    let authorization = Arc::new(AuthorizationProvider::new(settings.api_keys));
```

Comment:
if settings.api_keys.is_empty() {tracing::warn!("no api keys found in settgins, please provide in order to bla bla bla")}